### ISSUE TITLE
Website a11y: no alt text/labels on home page

### ISSUE THREAD

**Issue by julianna-langston** [finos/perspective#2133]
### Description of Problem:
The homepage of the Perspective website doesn't provide alt text for any of its 75 images. This means that any blind, screen reader users who come to the page will miss out on half of the page's content (measured vertically).

(Additionally, the velocity range slider doesn't have a label.)

### Potential Solutions:
Provide alt text to the images. I would suggest adding a `title` or `alt` attribute to [the examples list](https://github.com/finos/perspective/blob/master/docs/src/components/ExampleGallery/features.js), and then pass that property along to the homepage's `img` tag. [I believe that code exists here](https://github.com/finos/perspective/blob/master/docs/src/components/ExampleGallery/index.js#L27).

For the range slider, if you update [the input field here](https://github.com/finos/perspective/blob/master/docs/src/components/Demo/index.js#L116) to have an attribute like `aria-label="Velocity"` or `aria-label="Demo update rate in messages per second"`.

I'm willing to implement this change.

**Comment by texodus** [comment]
> I'm willing to implement this change.

Go for it, we'd merge this change.
